wake-lock/wakelock-screen-type-on-worker.https.worker.js

```diff
@@ -0,0 +1,8 @@
+//META: title=Screen wake lock should not be allowed in dedicated worker
+importScripts("/resources/testharness.js");
+
+promise_test(async t => {
+  await promise_rejects(t, "NotAllowedError", WakeLock.request('screen'));
```

Comment:
I think `async` and `await` are unnecessary here: simply `return`ing  `promise_rejects()` also works.

Reply:
Done. :)